app/src/main/java/com/example/minwoo/pingpongchat_client/LoginActivity.java

```diff
@@ -74,18 +118,29 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 String personName = account.getDisplayName();
                 String personEmail = account.getEmail();
                 String personId = account.getId();
+                String personIdToken = account.getIdToken();
 
                 Log.e("GoogleLogin", "personName=" + personName);
                 Log.e("GoogleLogin", "personEmail=" + personEmail);
                 Log.e("GoogleLogin", "personId=" + personId);
+                Log.e("GoogleLogin", "personIdToken=" + personIdToken);
             } catch (ApiException e) {
             }
         }
     }
 
-    private void signIn() {
-        Intent signInIntent = mGoogleSignInClient.getSignInIntent();
-        startActivityForResult(signInIntent, RC_SIGN_IN);
+    private void handleSignInResultOnStart(GoogleSignInResult result) {
+        Log.d("TAG", "handleSignInResult:" + result.isSuccess());
+        if (result.isSuccess()) {
+            // Signed in successfully, show authenticated UI.
+
+            Toast.makeText(LoginActivity.this, "로그인 성공", Toast.LENGTH_SHORT).show();
+            Intent loginIntent = new Intent(LoginActivity.this, MainActivity.class);
+            LoginActivity.this.startActivity(loginIntent);
+        } else {
+            // Signed out, show unauthenticated UI.
+            Toast.makeText(LoginActivity.this, "로그인 실패", Toast.LENGTH_SHORT).show();
+        }
     }
 
     private void handleSignInResult(Task<GoogleSignInAccount> completedTask) {
```

Comment:
it seems like this method NOT USED.
if you don't need this method anymore please remove it.